subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CacheTaskArchiveErrorIntegrationTest.groovy

```diff
@@ -223,8 +219,8 @@ class CacheTaskArchiveErrorIntegrationTest extends AbstractIntegrationSpec {
         errorOutput.contains("Couldn't read content of file '${link}'")
     }
 
-    private TestFile cleanBuildDir() {
-        file("build").deleteDir()
+    private void cleanBuildDir() {
+        run(":clean")
     }
```

Comment:
I wonder if this is a good way to fix the tests. Shouldn't we detect the `build` directory being removed too?

Also, the method name should be something different I think. Or perhaps it should be inlined.